src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
   margin: 0;
 }
+
+.header {
+  width: 100%;
+  display: flex;
+  background-color: white;
+  align-items: center;
+  justify-content: space-between;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  flex-shrink: 0;
+  margin: auto 50px;
+}
+
+.nav {
+  margin: 0 50px 0 0;
+}
+
+.nav__list {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  padding: 0;
+  margin: 0;
+  height: 60px;
+  align-items: center;
+}
+
+.nav__link {
+  text-decoration: none;
+  color: black;
+  font-size: 12px;
+  font-style: normal;
```

Comment:
Default value, so you can just remove it
```suggestion
```

Reply:
done